### ISSUE TITLE
Barcode scanner support.

### ISSUE THREAD

**Issue by ulbricht-inr** [hftlclub/ng-iltis#32]
A other nice to have :D... makes no senso for beer-boxes, but if you want to checkout 3 bottles of pfeffi and 2 havanna, it will be much faster.. if a barcode scanner is mounted on the stock-door.

**Comment by fmalcher** [comment]
Nice idea :-) I got three of them in stock. However, it might be difficult wo connect them with the mobile device

**Comment by ulbricht-inr** [comment]
USB...its a keyboard which enters numbers and pressed enter....
but connecting them to the mobile device, maked them less mobile.. better an extra device RPI+barcodescanner mounted fixed on the wall....hm.. maybe you should move this to the backend :D

**Comment by fmalcher** [comment]
Yeah, that's my point ;-) however, we can think about installing one of those scanners and connecting to the front-end via websocket.
